### PR TITLE
Conductance tables

### PR DESCRIPTION
**Pull request by pbmanis**
Largely completed implementation of the conductance tables. All channel parameters that were set in "species scaling" routines for the individual cell types were moved into a data table "ionchannels.py". Verified that tests on individual cells/model types yielded the same results as previous version. 

Added some mouse models (_cartwheel, tuberculoventral, mouse bushy, t and d stellate, sgcs_) to the test routines as well. 

Cleaned up some other sections and comments.

Propose merging this and the "fixes_Oct 2017" branches into the master branch. 

A "to-do" list:

*  to clean up the decorator routines by putting the scaling factors for different compartment types, and spatial distributions along processes, into data tables as well. At present this
* Allow data tables to be read from a different location (current location as default; other locations for experimental runs).
* Wrap run code in all examples and save the run results along with the tables used in those runs.
* Finish tables for other models (Spencer Octopus, Eager T-stellate).

### CODE REVIEW

File: cnmodel/data/_db.py [hunk 1]
```diff
@@ -35,12 +35,12 @@ def _lookup(ind, *args, **kwds):
             data[k] = DATA[key][ind]
         return data
     else:
-            return DATA[key][ind]
+        return DATA[key][ind]
             
 def setval(val, *args, **kwds):
     key = mk_key(*args, **kwds)
-    if key in DATA:
-        raise RuntimeError("Data key '%s' has already been set." % str(key))
```
Comment (campagnola) [inline cnmodel/data/_db.py]: Why remove this?

File: cnmodel/data/ionchannels.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+
+add_table_data('TV_channels', row_key='field', col_key='cell_type', 
+               species='mouse', data=u"""
+
+This table describes the ion channel densities and voltage shifts
+for a mouse tuberculoventral cell model.
+Ad-hoc model, based on the t-stellate cell model, but adjusted
+to match the data from Kuo and Trussell.
+
+            self.set_soma_size_from_Cm(35.0)
```
Comment (campagnola) [inline cnmodel/data/ionchannels.py]: I guess you meant to remove this.

**Review by campagnola — COMMENTED** [review]
Looks great! The new tables are very nice.

File: cnmodel/data/_db.py (hunk 1 above)
Comment (pbmanis) [reply cnmodel/data/_db.py]: If it is left in, we get an error:
File "/Users/pbmanis/Desktop/Python/cnmodel/cnmodel/data/_db.py", line 43, in setval
    raise RuntimeError("Data key '%s' has already been set." % str(key))

I am investigating this, will report back.

File: cnmodel/data/ionchannels.py (hunk 2 above)
Comment (pbmanis) [reply cnmodel/data/ionchannels.py]: Yes, and I have removed this. It will be corrected in the next PR.

File: cnmodel/data/_db.py (hunk 1 above)
Comment (pbmanis) [reply cnmodel/data/_db.py]: Error in XM13_channels table header (duplicate header). Fixed table, will reinstate code as check for that error.

**Comment by pbmanis** [comment]
Passes all tests. Ready to commit and move on. Any additional comments?

**Pull request merged**